src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
But `opt_callback` is actually optional. `scheduleChangeSize_` can be called either from `attemptChangeSize` or `changeSize` in `resourses.impl.js` If it is call from `changeSize there's no guarantee there will be a callback func